playbooks/robusta_playbooks/deployment_restart.py

```diff
@@ -0,0 +1,34 @@
+from robusta.api import *
+import datetime
+import logging
+
+
+class DeploymentParams(RateLimitParams):
+    """
+    :var name: name of deployment that needs to restart.
+    :var namespace: namespace of deployment that needs restart.
+    """
+    name: str
+    namespace: str
+
+
+@action
+def deployment_restart(alert: PrometheusKubernetesAlert, params: DeploymentParams):
```

Comment:
Do you mind naming it `deployment_restart_by_name`.
We might add another action in the future, that will do the same but using the deployment in the alert (according to labels). So we want to save a name like this for that one